README.rst

```diff
@@ -42,6 +42,7 @@ NOTE: The current master branch is now Python 3 only.  Python 2 support is
 being dropped.
 
 `Home page <http://matplotlib.org/>`_
+`Home page for devdocs <https://dev.matplotlib.org/>`_
```

Comment:
`devdocs` is slang. Something like 
```suggestion
`Home page based on master development branch <https://dev.matplotlib.org/>`_
```
is more clear. Feel free to improve on the wording.